pkg/webhook/config.go

```diff
@@ -25,14 +25,27 @@ import (
 // Config defines the set over environment variables required
 // for running this application.
 type Config struct {
-	Port          string `env:"PORT,default=8080"`
-	ProjectID     string `env:"PROJECT_ID,required"`
-	TopicID       string `env:"TOPIC_ID,required"`
-	WebhookSecret string `env:"WEBHOOK_SECRET,required"`
+	BigQueryID           string `env:"BIG_QUERY_ID,required"`
+	DatasetID            string `env:"DATASET_ID,required"`
```

Comment:
Why are these skipped in the validate function if they are required?

Reply:
just overlooked it, fixing